l1-contracts/src/core/libraries/rollup/RewardLib.sol

```diff
@@ -183,22 +183,29 @@ library RewardLib {
         $er.rewards += (blockRewardsAvailable - sequencerShare).toUint128();
       }
 
+      bool isIgnition = FeeLib.getManaTarget() == 0;
```

Comment:
I'm not completely sure about how fees work, but shouldn't this be `getManaLimit` instead?

Reply:
I'd also push for renaming this to `manaUsageEnabled` or `txsEnabled` or similar

Reply:
I can extend FeeLib to have a function like `isTxsEnabled`? Making it a bit more consistent across.